ryan-meinzer.txt

```diff
@@ -0,0 +1,91 @@
+https://codepen.io/ryanmeinzer/pen/rNarWbb
+
+    1. What is Semantic HTML? 
+    semantic = meaning
+    2. What is HTML used for? 
+    structuring language for the web and apps to grow
+    3. What is an attribute and where do we put it? 
+    it's a property that can change of a component, putting in html for example as a characteristic of a page element
+    4. What is the h1 tag used for? How many times should I use it on a page?
+    h1 is the main hearder tag - can use on a page for the main heading but may not be practical to use it more than once
+    5. Name two tags that have required attributes
+    img and href
+    6. What do we put in the head of our HTML document? 
+    things that we want the browser to know/see but don't want the end user to see
+    7. What is an id?
+    a unique reference for an element (such as a section of a page) 
+    8. What elements can I add an id to? 
+    text
+    9. How many times can I use the same id on a page? 
+    once
+    10. What is a class? 
+    a prototype from which objects are created
+    11. What elements can I add a class to? 
+    objects
+    12. How many times can I use the same class on a page? 
+    infinite
+    13. How do I get my link to open in a new tab?
+    set the target attribute to "_blank"
+    14. What is the alt attribute used for? 
+    alternative text description for an image
+    15. How do I reference an id?
+    #
+    16. What is the difference between a section and a div
+    a div typically holds sections which are areas of a page
```

Comment:
This is true, but sections can hold divs as well.  They're both container block elements but **section** is semantic whereas **div** is not

Reply:
ah gotcha, thanks